### PR TITLE
Add ability to link/unlink multiple students and multiple lessons

### PR DESCRIPTION
**Pull request by wpinrui**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -444,7 +444,8 @@ Examples:
 * `list` displays all students and lessons in TutorAid by only showing their names and list indexes.
 * `list -a` displays all students and lessons in TutorAid while showing all of their fields' data.
 
-### Adding a student to a lesson: `add -sl`
+
+### Adding student(s) to lesson(s): `add -sl`
 {:.no_toc}
 Tells TutorAid that a student attends a particular lesson.
```
Comment (JunWei3112) [inline docs/UserGuide.md:451]: A few nitpicks on this:
* Need to update the command format to be `add -sl s/STUDENT_INDEX... l/LESSON_INDEX...` instead.
* The line `Adds the student at the specified student index to the lesson at the specified lesson index.` can be changed to `Adds the students at the specified student indexes to the lessons at the specified lesson indexes.`
* The example can perhaps be changed to "`add -sl s/1 2 3 l/2` adds the students with indexes 1, 2 and 3 into the lesson with index 2."

These issues apply to the `del -sl` command too

File: src/main/java/tutoraid/logic/parser/AddStudentToLessonCommandParser.java [hunk 2]
```diff
@@ -15,8 +17,8 @@
 public class AddStudentToLessonCommandParser implements Parser<AddStudentToLessonCommand> {
 
     /**
-     * Parses the given {@code String} of arguments in the context of AddStudentToLessonCommand
-     * and returns an AddStudentToLessonCommand object for execution.
+     * Parses the given {@code String} of arguments in the context of AddStudentsToLessonsCommand
+     * and returns an AddStudentsToLessonsCommand object for execution.
```
Comment (JunWei3112) [inline src/main/java/tutoraid/logic/parser/AddStudentToLessonCommandParser.java:21]: `AddStudentsToLessonsCommand` should be `AddStudentToLessonCommand` instead, as seen from the return type

**Review by JunWei3112 — CHANGES_REQUESTED** [review]
Just a few minor issues. Else, the implemented functionality works well, LGTM!

**Review by JunWei3112 — APPROVED** [review]
LGTM

**Review by ErnestCuong — APPROVED** [review]
LGTM

**Pull request merged**